# Service to set a runtime parameter

RuntimeParameter param
---
RuntimeParameter actual